src/azure-cli/HISTORY.rst

```diff
@@ -4,7 +4,9 @@ Release History
 ===============
 
 **AppService**
+
 az webapp create : Fixing issue when running the command with --runtime 
+functionapp: Added error message to deployment command if resource group/function name invalid
```

Comment:
from last release, history.rst is automatically generated, no need to manual update it. But detail change is added in PR title. Guideline here: https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests